01week/javascripting/introduction.js

```diff
@@ -1 +1,74 @@
-console.log('hello');
+'use strict';
+
+/* return date program */
+function returnDate(){
+  const date = new Date();
+  return date;
```

Comment:
Can be simplified into a one line function, the const is redundant. 
`return new Date()`